docs/manual/manual_ifci_control.rst

```diff
@@ -7,23 +7,87 @@
 
-Vertiq modules can be controlled with the IQUART Flight Controller Interface (IFCI) while running pulsing firmware. A specifc IQUART packet has been created to allow for highspeed control of Vertiq modules. A single packet can include all of the control signals for multiple modules connected to the same UART port on a flight controller. Additionally a telemetry packet can be requested from a specific module every time the packet is sent. An example of a full IFCI packet is shown in the diagram below.
+About IFCI and Control Values
+=================================
+
+Vertiq modules may be controlled through the IQUART Flight Controller Interface (IFCI) when running applicable firmware. Please refer to your module's family page to see if 
+IFCI is supported. The IQUART Flight Controller Interface leverages Vertiq's :ref:`IQUART serial protocol <uart_messaging>` in order to send high speed controls to your modules. 
+These controls are packed into a single message allowing one IFCI packet to send controls to multiple modules connected to the same serial line. 
+The basic structure of an IFCI packet is illustrated below:
 
 .. figure:: ../_static/manual_images/pulsing_propeller/Long_Packed_Control_Message.png
     :align: center
     :alt: Packed Control Message Description
 
     Packed Control Message Description
 
-A general description of IQUART packets can be found in :ref:`uart_messaging`.
+Please note that an IFCI packet does not necessarily need to populate all 16 control values.
+
+Highlighted in orange are the bytes that are necessary in order to send valid IQUART messages, and are not specific to IFCI control.
+
+Highlighted in green are Control Values. A control value is an unsigned 16 bit value that defines a throttle, an X, or a Y command. X and Y commands are only applicable when 
+using Vertiq's :ref:`pulsing firmware <pulsing_module_start_guide>`. Any type of command can occupy any control value, and the location it occupies in an IFCI packet is called its Control 
+Value Index (CVI). For example, suppose you want to control a generic quadcopter. You may place the throttle commands for motor 1 in CVI 0, commands for motor 
+2 in CVI 1, and so on. Since the control values' order is fully configurable, however, it would be equally valid to place motor 4's throttle command in CVI 0.
+
+Having the ability to place any control value in any index also means that you must configure your Vertiq module to know which control value indices to read for its 
+specific commands. For example, suppose your flight controller is configured to transmit 10 control values. The throttle command for the module being 
+configured is the 5th command (CVI 4) in the IFCI packet. In order to have the module listen to that command, you simply set its `Throttle CVI` to 4. 
+The same principle applies to X and Y commands as well.
+
+.. note::
+    If any module CVI is set to 255, the module will not look in any received control values for a command.
```

Comment:
This is phrased a bit weirdly for a couple of reasons:

- The phrase, "module CVI" is a bit funky, since module's can have multiple CVIs they care about it when dealing with pulsing. It implies a lot of singularness.
- Also the "not look in any received control values" feels weird, using look just feels a bit strange to me personally.

I might say something along the lines of "If the CVI for any of the control values needed by a module is set to 255, then the module will not attempt to extract the control value from any index in the IFCI packet." or something like that.